logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/team/SyncSelfTeamUseCase.kt

```diff
@@ -41,14 +41,11 @@ internal class SyncSelfTeamUseCaseImpl(
         val user = userRepository.observeSelfUser().first()
 
         return user.teamId?.let { teamId ->
-            teamRepository.fetchTeamById(teamId = teamId).flatMap {
-                teamRepository.fetchMembersByTeamId(
-                    teamId = teamId,
-                    userDomain = user.id.domain
-                )
-            }.onSuccess {
-                teamRepository.syncServices(teamId = teamId)
-            }
+            teamRepository.fetchTeamById(teamId = teamId)
+                .map { }
+                .onSuccess {
+                    teamRepository.syncServices(teamId = teamId)
```

Comment:
we still need to fetch the full list of services since those are not included in search result